analyzers/tests/SonarAnalyzer.TestFramework/Verification/Verifier.cs

```diff
@@ -363,5 +369,23 @@ private void ValidateCodeFix()
             && (extension.Equals(".razor", StringComparison.OrdinalIgnoreCase) || extension.Equals(".cshtml", StringComparison.OrdinalIgnoreCase));
 
         public sealed record CompilationData(Compilation Compilation, string[] AdditionalSourceFiles);
+
+        private static int RestorePackages(string path, string workingDirectory)
+        {
+            using var process = new Process();
+            process.StartInfo = new ProcessStartInfo
+                                {
+                                    FileName = GetDotnetFilePath(),
+                                    Arguments = $"restore \"{path}\"",
+                                    WorkingDirectory = workingDirectory
+                                };
```

Comment:
This should be +4 from beginning of previous line
```suggestion
                    {
                        FileName = GetDotnetFilePath(),
                        Arguments = $"restore \"{path}\"",
                        WorkingDirectory = workingDirectory
                    };
```